integration-tests/src/tests/client/features/zero_balance_account.rs

```diff
@@ -305,3 +306,39 @@ fn test_zero_balance_account_upgrade() {
     let outcome = env.clients[0].chain.get_final_transaction_result(&tx_hash2).unwrap();
     assert_matches!(outcome.status, FinalExecutionStatus::SuccessValue(_));
 }
+
+#[test]
+fn test_storage_usage_components() {
+    // confirm these numbers don't change, as the zero balance limit is derived from them
+    const PUBLIC_KEY_STORAGE_USAGE: usize = 33;
+    const FULL_ACCESS_PERMISSION_STORAGE_USAGE: usize = 9;
+    const FUNCTION_ACCESS_PERMISSION_STORAGE_USAGE: usize = 98;
```

Comment:
Let's rename to `FULL_ACCESS_KEY_STORAGE_USAGE` and `FUNCTION_ACCESS_KEY_STORAGE_USAGE`, respectively. It also accounts for storing `Nonce` which is 8 bytes.